sdk/keyvault/azure_security_keyvault_keys/README.md

```diff
@@ -0,0 +1,61 @@
+# Azure KeyVault Keys client library for Rust
+
+Azure Key Vault is a cloud service that provides secure storage and management of sensitive information such as API keys, passwords, certificates, and cryptographic keys. The `azure_security_keyvault_keys` crate provides a client library for interacting with the Azure Key Vault Keys service.
+
+## Getting started
+
+### Install the crate
+
+Add the following to your `Cargo.toml` file:
+
+```toml
+[dependencies]
+azure_security_keyvault_keys = "0.1"
+```
+
+### Prerequisites
+
+- An [Azure subscription](https://azure.microsoft.com/free/)
+- A [Key Vault resource](https://docs.microsoft.com/azure/key-vault/quick-create-portal)
+
+### Authentication
+
+This crate uses the [azure_identity](https://crates.io/crates/azure_identity) crate for authentication. You can authenticate using a variety of methods, including environment variables, managed identity, and more.
+
+## Key concepts
+
+### KeyClient
+
+The `KeyClient` struct provides methods to manage keys in the Azure Key Vault. You can create, retrieve, update, and delete keys, as well as perform cryptographic operations.
+
+## Examples
+
+### Creating a key
+
+```rust
+
+#[tokio::main]
+async fn main() -> azure_core::Result<()> {
+    let credential = DefaultAzureCredential::new().unwrap();
+
+    let mut options = KeyClientOptions::default();
+
+    let client = KeyClient::new(
+        "https://<your-key-vault-name>.vault.azure.net",
+        credential.clone(),
+        Some(options));
+
+    let key = client.create_key("my-key", None).await?.into_body().await?;
+    println!("Created key: {:?}", key);
+
+    Ok(())
+}
+```
```

Comment:
```suggestion
```rust no_run
use azure_identity::DefaultAzureCredential;
use azure_security_keyvault_keys::{models::KeyCreateParameters, KeyClient};

#[tokio::main]
async fn main() -> Result<(), Box<dyn std::error::Error>> {
    let credential = DefaultAzureCredential::new()?;

    let client = KeyClient::new(
        "https://<your-key-vault-name>.vault.azure.net",
        credential.clone(),
        None,
    )?;

    let key = client
        .create_key("my-key", KeyCreateParameters::default().try_into()?, None)
        .await?
        .into_body()
        .await?;
    
    println!("Created key: {:?}", key);

    Ok(())
}
```
```

Reply:
Matching secrets example format and adding imports and main returns a `Result<(), Box<dyn std::error::Error>>`